ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/radiobutton/v1/radiobutton/radiobutton.html

```diff
@@ -48,7 +48,7 @@
                         value="${item.toString}"
                         disabled="${!radioButton.enabled || radioButton.readOnly}"
                         checked="${radioButton.enums[itemList.index] == radioButton.default[0]}"
-                        aria-describedby="${radioButton.id}__errormessage ${radioButton.id}__longdescription ${radioButton.id}__shortdescription"/>
```

Comment:
why are we removing this?

Reply:
aria-describedby is applied on the parent div of the radio-button options